### PR TITLE
Added Music

### PR DESCRIPTION
**Pull request by naveentherocker**
- Added Background Music to the Game
- Added Audio to the Tile

Ping @niccokunzmann

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Thanks!

One remark:
We still need the license files close the the music files. 
https://github.com/fossasia/labyrinth#how-to-add-music
Also see the different files and licenses added as an example.

File: js/sound.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ const backgroundAudio = [
         legalNotice : "Music by bensound.com",
         link : "http://bensound.com/",
     },
+    {
+        filename : "audio/music/perfect.mp3",
+        backgroundSongName: "Perfect",
+        author: "Ed Sheeran",
```
Comment (0x48piraj) [inline js/sound.js]: Ping @niccokunzmann :open_mouth: 

What's this ??
We can't use this song !

**Comment by niccokunzmann** [comment]
@0x48piraj Please create a revert of this, if you think this should not be in the game.

**Comment by naveentherocker** [comment]
@niccokunzmann @0x48piraj This song has no Copyright Issues.

**Comment by 0x48piraj** [comment]
Are you serious right now ?
I'm talking about Ed Sheeran's Perfect !

> https://www.youtube.com/watch?v=2Vv-BfVoq4g

**Pull request merged**